packages/commerce-sdk-react/README.md

```diff
@@ -64,7 +101,259 @@ const App = ({children}) => {
 
 export default App
 ```
-### Useful Links:
+
+## Shopper Authentication and Token Management
+
+_💡 This section assumes you have read and completed the [Authorization for Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/authorization-for-shopper-apis.html) guide._
+
+To help reduce boilerplate code for managing shopper authentication, by default, this library automatically initializes shopper session and manages the tokens for developers. Currently, the library supports the [Public Client login flow](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-public-client.html).
+
+### Shopper Session Initialization
+
+On `CommerceApiProvider` mount, the provider initializes shopper session by initiating the SLAS Public Client login flow. The tokens are stored/managed by the library to support Phased Launch use cases and multi-site setup.
+
+When the access token is expired, the library will automatically try to refresh the access token by calling the `/oauth2/token` endpoint with `grant_type=refresh_token`.
```

Comment:
I feel like there is too many implementation details here. I think we need to shorten this up and only talk about what might be important to the end user. I think one of those important things is where tokens are stored and what names they are store under. And whether or not those names can be changed or not (I can't remember is that is something that is configurable or not).

Reply:
🤔 The token names and how we store them **were**  important to users because the implementation was in user land and developers had to understand it to update the code and consume new changes. Now, with the hook library, the tokens also became implementation details.

Reply:
I think it's important to note that the hook lib uses **public client** instead of a private client at the moment.

Reply:
I'll remove the details for refresh_token stuff